charts/incubator/kimai/templates/_configmap.tpl

```diff
@@ -9,15 +9,15 @@
 kimai-config:
   enabled: true
   data:
-    APP_ENV: prod
-    DB_TYPE: mysql
+    APP_ENV: "prod"
+    DB_TYPE: "mysql"
     DB_PORT: "3306"
-    DB_USER: {{ .Values.mariadb.mariadbDatabase }}
-    DB_BASE: {{ .Values.mariadb.mariadbUsername }}
+    DB_USER: "{{ .Values.mariadb.mariadbDatabase }}"
+    DB_BASE: "{{ .Values.mariadb.mariadbUsername }}"
     {{/* Admin credentials */}}
-    ADMINMAIL: {{ .Values.kimai.credentials.adminMail }}
-    ADMINPASS: {{ .Values.kimai.credentials.adminPass }}
+    ADMINMAIL: "{{ .Values.kimai.credentials.adminMail }}"
+    ADMINPASS: "{{ .Values.kimai.credentials.adminPass }}"
```

Comment:
```suggestion
    ADMINPASS: {{ .Values.kimai.credentials.adminPass | quote }}
```

You can remove quotes from the rest.
Only things than can be numbers or booleans need to be quoted!
Frankly password shouldn't be either. but eh